app/templates/home/home.html

```diff
@@ -1,45 +1,124 @@
-{% extends "base.html" %}
-
-{% block heading %}
-    <h1>Home page</h1>
-{% endblock %}
-
-{% block content %}
-<button id="filterPanel" class="btn btn-primary" type="button" data-bs-toggle="collapse" data-bs-target="#collapseFilter"
-    aria-expanded="false" aria-controls="collapseFilter">
-    Open Filter Panel
+{% extends "base.html" %} {% block heading %}
```

Comment:
Why did you collapse all the tags below? It looks a bit squished together now. Could you please put this back as it was.

Reply:
We need to agree how those search fields will be arranged. I simply put my own below the existing ones, I did comment that there is need to harmonize it. That can happen even after the functionality is fully tested